### PR TITLE
product-cards

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
- [DEMO LINK](https://SergeyShapovalenko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://SergeyShapovalenko.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+      href="https://fonts.gstatic.com" 
+      crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card_logo" 
+        src="images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
```
Comment (tetiana-ivanchenko) [inline src/index.html:22]:
```suggestion
      <img 
        class="card_logo" 
        src="images/imac.jpeg" 
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <p class="stars_reviews">Reviews: 5</p>
+      </div>
+      <p class="stars_price">Price: <span class="price_numbers">$2,199</span></p>
+      <a 
+        class="stars_button" 
+        href="BUY" 
+        data-qa="hover">
```
Comment (tetiana-ivanchenko) [inline src/index.html:37]:
```suggestion
      <a 
        class="stars_button" 
        href="BUY" 
        data-qa="hover"
      >
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+.card_logo {
+  width: 160px;
+  height: 134px;
+
+  margin: 33px 20px 0;
+}
+
+.card_name {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/style.css:27]: redundant

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  width: 166px;
+  height: 36px;
+
+  margin: 40px 16px 0;
+  
+  color: #060b35;
+}
+
+.card_code {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/style.css:41]: the same

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+  font-style: normal;
```
Comment (tetiana-ivanchenko) [inline src/style.css:42]: normal - is default value

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.stars_reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding-left: 17px;
+  
+
+  margin: 0;
```
Comment (tetiana-ivanchenko) [inline src/style.css:79]: did you need so many empty lines?

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+.stars_button {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/style.css:134]: remove

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix all indentation (fix test failed in test-report)

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,44 @@
...
+        <div class="stars__star"></div>
+        <div class="stars_reviews">Reviews: 5</div>
+      </div>
+      <p class="stars_price">Price:<span class="price_numbers">$2,199</span></p>
+      <a 
+        class="stars_button" 
+        href="BUY" 
+        data-qa="hover"
+        >
+        BUY
```
Comment (loralevitska) [inline src/index.html:40]:
```suggestion
      <a 
        class="stars_button" 
        href="BUY" 
        data-qa="hover"
      >
        BUY
```

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+.stars__star:nth-child(-n+2),
+.stars__star:nth-child(-n+3),
+.stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  
+  width: 16px ;
+  height: 16px;
+ 
+  margin-right: 4px;
```
Comment (loralevitska) [inline src/style.css:92]: Remove repeatable code

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi)
Fix all previous  and new comments
Try redeploy you changes. Now your link doesn't work
image
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

File: src/style.css (hunk 9 above)
Comment (SergeyShapovalenko) [reply src/style.css]: Hello, all tests passed. Links doesn't work because of problems in GitHub.

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+  color: #fff;
+  
+  margin: 16px 16px 0;
+}
+
+.stars_button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  
+  color: #00acdc;
```
Comment (ViktorHavryliuk) [inline src/style.css:141]: Move repetable colors to variables

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  width: 200px;
+  height: 406px;
```
Comment (ViktorHavryliuk) [inline src/style.css:16]: Checklist p.8, don't hardcode height, content should dictate it
image

**Review by ViktorHavryliuk — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**